chainerio/filesystems/posix.py

```diff
@@ -1,10 +1,18 @@
 from chainerio.filesystem import FileSystem
+from chainerio.io import FileStat
 from chainerio.io import open_wrapper
 import io
 import os
 import shutil
 
 
+class PosixFileStat(FileStat):
+    inode = None
+    device = None
+    n_link = None
+    created = None
+
```

Comment:
Are you missing some fields here? `mode`, `owner`, `group`, `last_accessed`

Reply:
fixed in the latest commit! (3rd)